predicators/datasets/generate_atom_trajs_with_vlm.py

```diff
@@ -690,8 +719,29 @@ def _generate_ground_atoms_with_vlm_pure_visual_preds(
             image_option_trajs, vlm, 1)
         logging.info("Done querying VLM for candidate atoms!")
         # We now parse and sanitize this set of atoms.
+        atom_proposals_set = set([
+            "Cooked(patty)", "Raw(patty)", "IsBrown(patty)", "IsPink(patty)",
+            "IsGrilled(patty)", "Cut(lettuce)", "Diced(lettuce)",
+            "Sliced(lettuce)", "Whole(lettuce)", "Shredded(lettuce)"
+        ])
```

Comment:
this feels dangerous; could you introduce a command-line flag - something like `--use_hardcoded_vlm_atom_proposals` - and do something like how we do the debug grammar? I don't want this/things like this accidentally left-in such that they might trigger unexpected/silent failures in other envs.

Reply:
I was gonna remove this before the final PR -- but I left it in while I keep testing the PR.

But your proposal is better, will do that.